pygments/styles/nord.py

```diff
@@ -0,0 +1,74 @@
+"""
+    pygments.styles.nord
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    pygments version of the "nord" theme by Arctic Ice Studio
+    https://www.nordtheme.com/
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD-3 Clause License. See LICENSE for details
+"""
+
+from pygments.style import Style
+from pygments.token import Keyword, Name, Comment, String, Error, \
+     Number, Operator, Generic, Whitespace, Punctuation
+
+
+class NordStyle(Style):
+    """
+    Pygments version of the "nord" theme by Arctic Ice Studio
+    """
+
+    background_color = '#2e3440'
+    default = '#d8dee9'
+
```

Comment:
Can you please add the various `linenos` styles and the `highlight` color as well?